go.sum

```diff
@@ -1,9 +1,13 @@
+github.com/frankban/quicktest v1.2.2/go.mod h1:Qh/WofXFeiAFII1aEBu529AtJo6Zg2VHscnEsbBnJ20=
```

Comment:
I am not sure what this is?...

We do not use github.com/frankban/quicktest or github.com/google/go-cmp in this project... Are you sure this is correct?

Reply:
Yeah I'm not sure where those came from - maybe when I got a dependency wrong and then fixed it? I've removed them and rebuilt with no problems anyway.